extensions/wikia/CommunityPage/styles/components/_todo-list-module.scss

```diff
@@ -42,21 +42,20 @@ $module-header-line-height: 28px;
 		clear: both;
 	}
 
-	.community-page-todo-list-module-content {
-		border-style: solid;
+	.community-page-todo-list-module-content,
+	.community-page-todo-list-module-zerostate {
```

Comment:
waat? This can be done using @extend: https://css-tricks.com/the-extend-concept/